spyro/solvers/time_integration_central_difference.py

```diff
@@ -241,9 +242,12 @@ def central_difference_MMS(Wave_object, source_id=0):
 
     u_nm1 = Wave_object.u_nm1
     u_n = Wave_object.u_n
+    u_np1 = Wave_object.u_np1
     u_nm1.assign(Wave_object.analytical_solution(t - 2 * dt))
     u_n.assign(Wave_object.analytical_solution(t - dt))
-    u_np1 = fire.Function(Wave_object.function_space, name="pressure t +dt")
+    # u_np1 = fire.Function(Wave_object.function_space, name="pressure t +dt")
```

Comment:
I would remove commented code. They make code harder to read and make it harder to search for word occurrences.

Reply:
Removed